src/routes.js

```diff
@@ -12,6 +12,10 @@ function routesConfig($stateProvider, $urlRouterProvider, $locationProvider) {
     })
     .state('deck', {
       url: '/deck/:id',
-      template: `WIP`
+      template: `<deck-edit></deck-edit>`
+    })
+    .state('deckCreate', {
+      url: '/deckCreate',
```

Comment:
Does it mean that we can access the create deck form at the url `localhost:3000/deckCreate` ?